sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeEntitiesTests.cs

```diff
@@ -34,6 +34,24 @@ public class RecognizeEntitiesTests : TextAnalyticsClientLiveTestBase
             }
         };
 
+        [Test]
+        public async Task RecognizeEntitiesWithAADTest()
+        {
+            TextAnalyticsClient client = GetClient(useTokenCredential: true);
+            string document = singleEnglish;
+
+            CategorizedEntityCollection entities = await client.RecognizeEntitiesAsync(document);
+
+            Assert.AreEqual(3, entities.Count);
+
+            var entitiesList = new List<string> { "Bill Gates", "Microsoft", "Paul Allen" };
+            foreach (CategorizedEntity entity in entities)
+            {
+                Assert.IsTrue(entitiesList.Contains(entity.Text));
+                Assert.IsNotNull(entity.ConfidenceScore);
```

Comment:
This one is non-nullable, I believe.

Reply:
Hmm u are right. I will do the cleanup of the tests in issue https://github.com/Azure/azure-sdk-for-net/issues/14833 
so I will leave it as is for now